src/lib/shell/commands/Dns.cpp

```diff
@@ -83,15 +83,24 @@ class DnsShellResolverDelegate : public Dnssd::DiscoverNodeDelegate, public Addr
 
     AddressResolve::NodeLookupHandle & Handle() { return mSelfHandle; }
 
+    void LogOperationalNodeDiscovered(const Dnssd::OperationalNodeBrowseData & nodeData)
+    {
+        streamer_printf(streamer_get(), "DNS browse operational succeeded: \r\n");
+        streamer_printf(streamer_get(), "   Node ID: " ChipLogFormatX64 "-" ChipLogFormatX64 "\r\n",
```

Comment:
See comments above.

Reply:
Renamed `Node ID` but this going streamer and not chip logging, so cannot use `ChipLogFormatPeerId` and `ChipLogValuePeerId` here

Reply:
This seems to still be using "Node ID".

But also, I don't understand the "needs to go into streamer" distinction: `ChipLogFormatPeerId` is a macro that expands out exactly to `ChipLogFormatX64 "-" ChipLogFormatX64`, no?

Reply:
My bad, I got confused. I have updated.